### PR TITLE
R4R:  Add token-stats query by id

### PR DESCRIPTION
**Pull request by julia2804**
https://github.com/irisnet/irishub/issues/1579

### CODE REVIEW

File: client/bank/utils.go [hunk 1]
```diff
@@ -67,3 +67,25 @@ func (ts TokenStats) String() string {
...
+	BurnedToken sdk.Coin `json:"burned_token"`
+	BondedToken sdk.Coin `json:"bonded_token"`
+}
+
+// String implements fmt.Stringer
+func (ts TokenStatsOfAsset) String() string {
+	return fmt.Sprintf(`TokenStats:
+  Loose Token:  
+	denom:%v
+	amount:%v
```
Comment (chengwenxi) [inline client/bank/utils.go]: replace by coin.String()

File: docs/cli-client/bank/token-stats.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Query the token statistic, including total loose tokens, total burned token and
 
 | Name,shorthand | Type   | Required | Default               | Description                                                  |
 | -------------- | ------ | -------- | --------------------- | ------------------------------------------------------------ |
+| --id           |  String| False    |                       | id of asset                                                  |
```
Comment (chengwenxi) [inline docs/cli-client/bank/token-stats.md]: Don't need flag --id

File: tests/cli/bank_test.go [hunk 3]
```diff
@@ -75,3 +75,36 @@ func TestIrisCLIBankSend(t *testing.T) {
...
+	tests.WaitForNextNBlocksTM(2, port)
+
+	executeWrite(t, fmt.Sprintf("iriscli bank send %v --amount=10tttt-min --to=%s --from=foo --gas=10000 --fee=0.3iris", flags, barAddr), sdk.DefaultKeyPass)
+	tests.WaitForNextNBlocksTM(2, port)
+
+	_ = executeGetAccount(t, fmt.Sprintf("iriscli bank account %s %v", barAddr, flags))
+
+	tokenS := executeGetTokenStatsForAsset(t, fmt.Sprintf("iriscli bank token-stats %v tttt", flags))
+
+	fmt.Printf("tokenStats:%s\n", tokenS.String())
```
Comment (chengwenxi) [inline tests/cli/bank_test.go]: Add some result checks.

File: client/bank/lcd/query.go [hunk 4]
```diff
@@ -74,34 +74,90 @@ func QueryCoinTypeRequestHandlerFn(cdc *codec.Codec, cliCtx context.CLIContext)
...
+		vars := mux.Vars(r)
+		assetId := vars["id"]
 
-		var tokenStats bank.TokenStats
-		err = cdc.UnmarshalJSON(resToken, &tokenStats)
-		if err != nil {
-			utils.WriteErrorResponse(w, http.StatusInternalServerError, err.Error())
-			return
-		}
+		if len(assetId) == 0 || isIris(assetId) {
```
Comment (zhangyelong) [inline client/bank/lcd/query.go]: `id` is a required param of `/bank/token-stats/{id}`, it can not be routed here if the `id` is empty.

**Pull request merged**